lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1272,7 +1272,7 @@ protected function get_layout_styles( $block_metadata ) {
 		$has_block_gap_support    = _wp_array_get( $this->theme_json, array( 'settings', 'spacing', 'blockGap' ) ) !== null;
 		$has_fallback_gap_support = ! $has_block_gap_support; // This setting isn't useful yet: it exists as a placeholder for a future explicit fallback gap styles support.
 		$node                     = _wp_array_get( $this->theme_json, $block_metadata['path'], array() );
-		$layout_definitions       = _wp_array_get( $this->theme_json, array( 'settings', 'layout', 'definitions' ), array() );
+		$layout_definitions       = gutenberg_get_layout_definitions();
```

Comment:
We also need to remove `layout.definitions` from the `VALID_SETTINGS` in this class.

Reply:
What happened if 3rd parties provided `layout.definitions` via `theme.json`? The potential scenarios are: the theme via its `theme.json` and plugins via the [existing filters](https://make.wordpress.org/core/2022/10/10/filters-for-theme-json-data/).

Reply:
I found that `layout.definitions` were merged in core as of 6.1. PR https://github.com/WordPress/wordpress-develop/commit/72380e31ca0bc05dca5617357412ab8fd346a583 and trac ticket https://core.trac.wordpress.org/ticket/56467 If there wasn't any code to prevent them from being overridden by themes/plugins, we probably need to communicate this via a devnote stating it is a bug that others were able to override the definitions.

Reply:
Thanks! I've removed it from `layout.definitions`. If and when this PR lands, I'm happy to draft a dev note for 6.3 to flag the change 🙂